theme/fontcolor.css

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+:root {
+	--ck-table-color-tile-size: 20px;
+	--ck-table-color-max-columns: 5;
+	--ck-table-color-padding: 10px;
+
+	--ck-color-table-color-tile-border: hsl(0, 0%, 90%);
+	--ck-color-table-color-tile-border-active: hsl(184, 80%, 50%);
+}
+
+.ck .ck-color-table__grid-container {
+	display: grid;
+	grid-template-columns: repeat( var(--ck-table-color-max-columns), 1fr);
+	grid-gap: calc( var(--ck-table-color-padding) / 2 );
+	padding: var(--ck-table-color-padding);
+}
+
+.ck .ck-color-table__color-tile {
+	width: var(--ck-table-color-tile-size);
+	height: var(--ck-table-color-tile-size);
+	min-width: var(--ck-table-color-tile-size);
+	min-height: var(--ck-table-color-tile-size);
+	border-radius: var(--ck-border-radius);
+	cursor: pointer;
```

Comment:
This is a wrong cursor. A default one should be used instead. Pointer is reserved for links.